samples/src/Microsoft.Azure.IIoT.App/src/Services/Publisher.cs

```diff
@@ -68,8 +70,8 @@ public class Publisher {
         /// <param name="samplingInterval"></param>
         /// <param name="publishingInterval"></param>
         /// <returns>ErrorStatus</returns>
-        public async Task<bool> StartPublishingAsync(string endpointId, string nodeId, string displayName,
```

Comment:
Convention is async methods should end in "Async" for name.   It is ok - but please fix in future again.

Reply:
fixed